src/node_util.cc

```diff
@@ -292,10 +292,47 @@ static void GetCallSite(const FunctionCallbackInfo<Value>& args) {
 
     callsite_objects.push_back(obj);
   }
+}
+
+static void IsInsideNodeModules(const FunctionCallbackInfo<Value>& args) {
```

Comment:
We can add V8 fast api to this to speed things up.

Reply:
I don't know if it's safe to iterate over the stack in the fast calls - probably safer to leave it to a follow up to avoid regressions if it's not and isn't reproducible from the tests.